api.go

```diff
@@ -66,14 +66,27 @@ func NewPathApi(ipfspath string) (*HttpApi, error) {
 	return NewApi(a)
 }
 
+// NewAddrApi constructs new HttpApi by pulling api address.
+// Api address should be set like /ip4/127.0.0.1/tcp/5001.
+func NewAddrApi(apiAddr string) (*HttpApi, error) {
+	a, err := ma.NewMultiaddr(strings.TrimSpace(apiAddr))
+	if err != nil {
+		if os.IsNotExist(err) {
```

Comment:
This will never be true.

Reply:
if this check will never be true. 
Parsing the multiaddr then calling NewApi is very simple